hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AtttributeResolver.kt

```diff
@@ -0,0 +1,61 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import uk.gov.justice.digital.hmpps.prisonersearch.common.model.Prisoner
+import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.api.TypeMatcher
+import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.api.genericType
+import kotlin.reflect.KClass
+import kotlin.reflect.KProperty1
+import kotlin.reflect.full.memberProperties
+
+typealias Attributes = Map<String, KClass<*>>
+
+@Configuration
+class AttributeResolver {
+  @Bean
+  fun attributes(): Attributes = getAttributes(Prisoner::class)
+}
```

Comment:
All of the supported attributes are now available in a Bean so they can be used easily by different parts of the system.